heif_benchmark.py

```diff
@@ -0,0 +1,231 @@
+"""
+Benchmarks HEIF compression time and size (the ratio of the compressed image to the original image)
+on a given set of images (300 landing pad images captured from a flight test).
+
+Creates a folder with a compressed image for each quality setting to visually check the quality,
+as well as a .json with the test data and a .csv which provides a more human-friendly summary
+of the data.
+"""
+import gc
+import io
+import json
+import os
+import pathlib
+import time
+
+from PIL import Image
+from pillow_heif import register_heif_opener
+
+
+# Setting parameters
+FRAME_COUNT = 300  # Total number of frames
+FRAME_TO_SAVE = 69  # This frame is good, it has both landing pads in it
+INPUT_PATH = pathlib.Path("test_images", "Encode Test Dataset 2024")
+OUTPUT_PATH = pathlib.Path(f"log_{int(time.time())}")
+COMPRESS_TYPES = [0, 1, 2, 3, 4]  # There are 5 different compression algorithms, each are numbered
+INITIAL_COMPRESS_LEVEL = 1  # Compress level 0 is skipped because it is uncompressed
+MAX_COMPRESS_LEVEL = 6  # Although this maxes out at 9, it takes way too long (like 10s per image)
```

Comment:
The HEIF has similar settings to AVIF and not PNG. See (docs)[https://pillow-heif.readthedocs.io/en/latest/saving-images.html#save-parameters] and `main` branch for more info.